### PR TITLE
Finalize teams & subjects frontend

### PR DESCRIPTION
**Pull request by TheBestTvarynka**

### CODE REVIEW

File: backend/src/main/java/com/eshka/controller/AuthController.java [hunk 1]
```diff
@@ -30,4 +30,10 @@ public void register(@RequestBody RegisterDTO registerData) {
         this.authService.register(registerData);
     }
 
+    @GetMapping("/logout")
+    @ResponseStatus(HttpStatus.OK)
+    public void logout() {
+        System.out.println("Log Out");
```
Comment (KySo4oK) [inline backend/src/main/java/com/eshka/controller/AuthController.java]: unacceptable

File: backend/src/main/java/com/eshka/controller/TeamController.java [hunk 2]
```diff
@@ -31,6 +34,23 @@
                 HttpStatus.OK);
     }
 
+    @ApiOperation("get full team info")
+    @GetMapping("/{id}/full")
+    public ResponseEntity<TeamFullResponse> getFullInfo(@PathVariable(name = "id") String id) {
```
Comment (KySo4oK) [inline backend/src/main/java/com/eshka/controller/TeamController.java]: teamId is better name

File: backend/src/main/java/com/eshka/dto/response/TeamFullResponse.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    private String description;
+    private List<SubjectResponse> subjects;
+    private List<UserShortResponse> members;
+
+    public static TeamFullResponse fromTeam(Team team) {
+        return new TeamFullResponse(
+                team.getId(),
+                team.getName(),
+                team.getDescription(),
+                team.getSubjects().stream()
```
Comment (KySo4oK) [inline backend/src/main/java/com/eshka/dto/response/TeamFullResponse.java]: unacceptable for entity class. This Business Logic must be in Services or mapper

File: backend/src/main/java/com/eshka/mapper/TeamMapper.java [hunk 4]
```diff
@@ -13,4 +13,4 @@
     TeamResponse teamToTeamResponse(Team entity);
 
     Team teamRequestToTeam(TeamRequest request);
-}
\ No newline at end of file
+}
```
Comment (KySo4oK) [inline backend/src/main/java/com/eshka/mapper/TeamMapper.java:19]: what was changed?)

File: backend/src/main/java/com/eshka/service/impl/TeamServiceImpl.java [hunk 5]
```diff
@@ -19,9 +19,9 @@
 public class TeamServiceImpl implements TeamService {
     private final TeamRepository teamRepository;
     private final UserRepository userRepository;
+    // @Value("${HOST}")
```
Comment (KySo4oK) [inline backend/src/main/java/com/eshka/service/impl/TeamServiceImpl.java]: why u comment it?

**Review by KySo4oK — CHANGES_REQUESTED** [review]
What is happening when front-ender trying to wrote smt in back-end

File: backend/src/main/java/com/eshka/controller/TeamController.java (hunk 2 above)
Comment (TheBestTvarynka) [reply backend/src/main/java/com/eshka/controller/TeamController.java]: Okay, I'll fix it.

File: backend/src/main/java/com/eshka/controller/AuthController.java (hunk 1 above)
Comment (TheBestTvarynka) [reply backend/src/main/java/com/eshka/controller/AuthController.java]: uups

File: backend/src/main/java/com/eshka/mapper/TeamMapper.java (hunk 4 above)
Comment (TheBestTvarynka) [reply backend/src/main/java/com/eshka/mapper/TeamMapper.java:19]: I don't know(

File: backend/src/main/java/com/eshka/controller/AuthController.java [hunk 6]
```diff
@@ -30,4 +30,9 @@ public void register(@RequestBody RegisterDTO registerData) {
         this.authService.register(registerData);
     }
 
+    @GetMapping("/logout")
+    @ResponseStatus(HttpStatus.OK)
+    public void logout() {
```
Comment (KySo4oK) [inline backend/src/main/java/com/eshka/controller/AuthController.java:35]: empty method?)

**Review by KySo4oK — APPROVED** [review]

**Pull request merged**